frontend/src/components/chat/Contact.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react";
+
+import { getUser } from "../../services/ChatService";
+import UserLayout from "../layouts/UserLayout";
+
+export default function Contact({ chatRoom, onlineUsersId, currentUser }) {
+  const [contact, setContact] = useState();
+
+  useEffect(() => {
+    const contactId = chatRoom.members?.find(
+      (member) => member !== currentUser.uid
+    );
+
+    const fetchData = async () => {
+      const res = await getUser(contactId);
+      setContact(res);
+    };
```

Comment:
This function will create every useEffect work, move this declaration out of `useEffect`